internal/daysteps/daysteps.go

```diff
@@ -1,13 +1,58 @@
 package daysteps
 
+import (
+	"errors"
+	"fmt"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/Yandex-Practicum/tracker/internal/personaldata"
+	"github.com/Yandex-Practicum/tracker/internal/spentenergy"
+)
+
 type DaySteps struct {
-	// TODO: добавить поля
+	Steps    int
+	Duration time.Duration
+	personaldata.Personal
 }
 
+// Parse преобразует строку в структуру DaySteps.
+// Принимает строку с данными о количестве шагов и продолжительности дня, возвращает ошибку, если данные некорректны.
 func (ds *DaySteps) Parse(datastring string) (err error) {
-	// TODO: реализовать функцию
+	data := strings.Split(datastring, ",")
+	if len(data) != 2 {
+		return errors.New("invalid data string")
+	}
+	ds.Steps, err = strconv.Atoi(data[0])
+	if err != nil {
+		return errors.New("invalid steps")
+	}
+	ds.Duration, err = time.ParseDuration(data[1])
+	if err != nil {
+		return errors.New("invalid duration")
```

Comment:
Теряешь сообщение об ошибки
```suggestion
		return fmt.Errorf("conversion error: %w", err)
```